lib/server_health_check.rb

```diff
@@ -25,13 +25,17 @@ def redis!(host: nil, port: 6379)
 
   def active_record!
     begin
-      ActiveRecord::Base.connection
-      @results[:active_record] = OK
-      true
+      if ActiveRecord::Base.connection_pool.with_connection { |connection| connection.active? }
+        @results[:active_record] = OK
+        true
+      else
+        @results[:active_record] = "Failed: unable to connect to database"
+        false
+      end
     rescue NameError
```

Comment:
IMO if we're concerned that AR::Base isn't defined, we should simply return early instead of raising an error.

Alternatively, just declare ActiveRecord::Base outside the begin-block so that the NameError doesn't need to be explicitly handled.